lib/schema_checker.py

```diff
@@ -80,7 +80,7 @@ def check_usage_scenario(self, usage_scenario):
             },
 
             Optional("services"): {
-                Use(self.contains_no_invalid_chars): {
+                Use(self.contains_no_invalid_chars): Or (None, {
```

Comment:
I understand that you want to remove services in the resulting usage_scenario from the initial compose, but the schema checker runs after the merge.

I would argue if after the merge the services key is empty we should still fail ...?

Reply:
A very very good catch. I initially allowed None as a value and handled it all places in the runner but then decided to do the "easier" thing and just remove them.